docs/administration-guide/gaffer-deployment/gremlin.md

```diff
@@ -79,13 +79,14 @@ A full breakdown of the available properties is as follows:
 
 | Property Key | Description | Used in REST API |
 | --- | --- | --- |
-| `gremlin.graph` | The Tinkerpop graph class we should use for construction. | No |
+| `gremlin.graph` | The TinkerPop graph class we should use for construction. | No |
 | `gaffer.graphId` | The graph ID of the Tinkerpop graph. | No |
 | `gaffer.storeproperties` | The path to the store properties file. | No |
 | `gaffer.schemas` | The path to the directory containing the graph schema files. | No |
-| `gaffer.userId` | The default user ID for the Tinkerpop graph. | No (User is always set via the [`UserFactory`](../security/user-control.md).) |
+| `gaffer.userId` | The default user ID for the TinkerPop graph. | No (User is always set via the [`UserFactory`](../security/user-control.md).) |
 | `gaffer.dataAuths` | The default data auths for the user to specify what operations can be performed | No |
 | `gaffer.rest.timeout` | The timeout for gremlin queries submitted to the REST API in ms. Default is 2 mins if not specified. | Yes |
 | `gaffer.operation.options` | Default `Operation` options in the form `key:value` (this can be overridden per query see [here](../../user-guide/query/gremlin/custom-features.md)) | Yes |
-| `gaffer.elements.getalllimit` | The default limit for unseeded queries e.g. `g.V()`. | Yes |
+| `gaffer.elements.getlimit` | The default limit applied to get element operations called by TinkerPop e.g. `GetElements` or `GetAllElements`. | Yes |
 | `gaffer.elements.hasstepfilterstage` | The default stage to apply any `has()` steps e.g. `PRE_AGGREGATION` | Yes |
+| `gaffer.includeOrphanedVertices` | Should orphaned vertices be returned by default in a result, these are vertices on an edge that have no associated entity in the Gaffer graph. Queries will likely be slower if enabled. | Yes |
```

Comment:
might be worth mentioning that the default value is false as it isn't made super clear.